gpii/node_modules/lifecycleManager/test/js/LifecycleManagerTests.js

```diff
@@ -64,16 +64,62 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
         return togo;
     };
 
-    gpii.tests.lifecycleManager.buildSettingsHandlersEntry = function (settings) {
-        return {
+    /**
+     * Used by a bunch of the tests to avoid having to write out the almost-identical full set of lifecycle
+     * instructions for each test. Instead one can have a "base set" of lifecycle instructions
+     * that can be shared across multiple tests/multiple phases of a single test, and then use
+     * this function to extend base set to match the specific test/test phase.
+     *
+     * The function takes an unlimited number of arguments. The first argument is mandatory and must
+     * be a set of lifecycle instructions entries keyed by solution ids. While the system wont explode
+     * if entries for multiple solutions are provided, only the "first" solution entry will be
+     * extended. The remaining arguments will be fluid.extended onto the first argument.
+     *
+     * @currentInstructions {Object} a set of lifecycle instructions keyed by a solution id. For example,
```

Comment:
Fix up JSDocs here - it should (now) be @param {Object} currentInstructions - A set of etc.